src/JNIHelper.h

```diff
@@ -26,6 +29,17 @@
 
 #define JRUNTIMEEXCEPT_CTOR "(java/lang/String;)V"
 
+// FIXME: The location of these JARs should be specified using a config file.  We should also
+// be more flexible about release numbers.
+// Defines for classpath additions needed by liballuxio.  If the locations of these files
+// don't match in your environment, you'll either have to set the CLASSPATH env var, or modify
+// these defines to point to the correct location
+#define CLASSPATH_ALLUXIO_JAR "$HOME/alluxio/core/client/target/alluxio-core-client-jar-with-dependencies.jar"
```

Comment:
this is a bit ugly (needs recompilation) to add here in code compared to simply require an env var setting (which is essentially a config).